eng/performance/helix_sender.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.10-bullseye
```

Comment:
I would vote for `3.10-bookworm` (or maybe better `3.10-slim-bookworm`), to be on current Debian 12.

Reply:
I tested out slim-bookworm but it was causing too many issues with other dependencies such as wget so just using bookworm now.